docs/source/index.rst

```diff
@@ -25,6 +25,74 @@ then!
 .. literalinclude:: trustme-trio-example.py
 
 
+CLI reference
+=============
+
+**All options:**
+
+.. code-block:: console
+
+   $ python -m trustme --help
+   usage: trustme [-h] [-d DIR] [-i IDENTITIES [IDENTITIES ...]] [--common-name COMMON_NAME] [--key-size KEY_SIZE] [-q]
+
+   optional arguments:
+     -h, --help            show this help message and exit
+     -d DIR, --dir DIR     Directory where certificates and keys are written to. Defaults to cwd
+     -i IDENTITIES [IDENTITIES ...], --identities IDENTITIES [IDENTITIES ...]
+                            Identities for the certificate. Defaults to
+     --common-name COMMON_NAME
+                           Also sets the deprecated 'commonName' field for all relevant identities
```

Comment:
commonName can only be set to a single identity, the docstring doesn't reflect that well.

Reply:
Not sure what you mean here. I've ran the following and it succeeded:

```bash
python -m trustme -i 127.0.0.1 localhost --common-name example.org
```

Do you mean that the `--common-name` will actually only be set to the first identity? What phrasing would you suggest?

Reply:
Yeah the help string makes it seem like multiple can here set which isn't correct.

Reply:
Oh got it, yeah a single common name value can be passed.